route/src/io/pedestal/http/route.clj

```diff
@@ -121,12 +121,21 @@
                     key
                     (.append b c)))))))))
 
+(defn parse-param-map [m] (reduce
```

Comment:
This should build up a transient and potentially use `reduce-kv` to remove some overhead.